### ISSUE TITLE
history_ssrnet_mt.h5

### ISSUE THREAD

**Issue by hegc** [shamangary/FSA-Net#25]
Hi, what is history_ssrnet_mt.h5? the model is trained on SSR_net_ori_MT？

**Comment by shamangary** [comment]
It's the file for recorded training loss and validation loss.
